src/components/layout/library-footer.tsx

```diff
@@ -1,30 +1,35 @@
 import Link from "@/components/patterns/elements/drupal-link"
 import {ArrowRightIcon} from "@heroicons/react/20/solid"
 import Lockup from "@/components/patterns/elements/lockup/lockup"
-import {DrupalLinkButton} from "@/components/patterns/link"
 import FacebookIcon from "@/components/patterns/icons/FacebookIcon"
 import LinkedInIcon from "@/components/patterns/icons/LinkedInIcon"
 import TwitterIcon from "@/components/patterns/icons/TwitterIcon"
 import InstagramIcon from "@/components/patterns/icons/InstagramIcon"
 import YoutubeIcon from "@/components/patterns/icons/YoutubeIcon"
 import {ReactNode} from "react"
+import Image from "next/image"
 
 const LibraryFooter = () => {
   return (
-    <div className="bg-black-true pb-30 text-white">
-      <div className="bg-footer-sprinkles bg-repeat">
-        <svg viewBox="0 0 1500 100" aria-hidden={true} className="bg-gradient-to-t from-black-true to-transparent">
-          <path d="M1500,0 L1500,100 1200,100 1200,0" className="fill-black-true"></path>
+    <div className="rs-mt-6 bg-fog-light pb-30 text-black @container @12xl:mt-0">
+      <div>
+        <svg viewBox="0 0 1500 100" aria-hidden={true} className="bg-gradient-to-t from-fog-light to-transparent">
+          <path d="M1500,0 L1500,100 1200,100 1200,0" className="fill-fog-light"></path>
           <path d="M0,0 Q500,10 700,45 Q1200,130 1500,0" stroke="#fff" className="fill-white"></path>
         </svg>
       </div>
-      <div className="centered mx-auto mb-50">
-        <div className="mx-auto">
-          <Lockup whiteText />
+      <div className="centered relative mb-50 mt-25 flex flex-row justify-between gap-2xl @3xl:mt-50 @5xl:mt-0">
+        <div>
+          <Lockup />
+        </div>
+        <div className="absolute bottom-50 right-[-5rem] block @3xl:right-[-3rem] @5xl:bottom-0 @5xl:right-[2rem] @8xl:right-[8rem] @12xl:hidden">
+          <div className="relative h-[18.5rem] w-180 @5xl:h-[20.5rem] @5xl:w-200 @8xl:h-[28.4rem] @8xl:w-[27.7rem]">
+            <Image src="/footer-nerd-squirrel.png" alt="" fill />
```

Comment:
remember to use `sizes` prop when using the `fill` prop. do you need to use some classes like `object-contain` or anything?